src/System.Runtime.Extensions/tests/System/Math.cs

```diff
@@ -1441,6 +1441,145 @@ public static void Round_Double_Digits()
             Assert.Equal(double.NegativeInfinity, Math.Round(double.NegativeInfinity, 3, MidpointRounding.AwayFromZero));
         }
 
+        [Theory]
```

Comment:
You should be able to avoid repeating the inline data using `MemberData`...see elsewhere in repo for examples

Reply:
using MemberData now